docs/API.md

```diff
@@ -108,7 +108,7 @@ Default: `0777`
 
 Octal permission string specifying mode for any folders that need to be created for output folder.
 
-### gulp.task(name[, deps], fn)
+### gulp.task(name [, deps] | [, deps[, cb]] | [, cb])
```

Comment:
Okay, I can take those off. 

The `fn` is not in `[ ]` and yet later in the documentation it says it's optional, for example in `gulp.task('build', ['list','of','deps'])` so shouldn't `fn` be optional as well?